README.md

```diff
@@ -6,6 +6,15 @@
 [![image](https://img.shields.io/pypi/pyversions/ruff/0.4.2.svg)](https://pypi.python.org/pypi/ruff)
 [![Actions status](https://github.com/astral-sh/ruff-vscode/workflows/CI/badge.svg)](https://github.com/astral-sh/ruff-vscode/actions)
 
+> [!NOTE]
+>
+> **As of Ruff v0.4.5, Ruff ships with a built-in language server written in Rust: ⚡ `ruff server` ⚡**
```

Comment:
This is available in versions prior to `v0.4.5`, but since the previous versions have issues and/or missing features, I'm fine with making this the 'minimum version' 👍